workflow/workflow.go

```diff
@@ -635,3 +636,17 @@ func IsContinueAsNewError(err error) bool {
 func DataConverterWithoutDeadlockDetection(c converter.DataConverter) converter.DataConverter {
 	return internal.DataConverterWithoutDeadlockDetection(c)
 }
+
+// DeterministicKeys returns the keys of a map in deterministic (sorted) order. To be used in for
```

Comment:
Pick whether to call this `DeterministicKeys` or `SortedKeys`. I don't have a preference

Reply:
Oh yeah I meant to update everything to `DeterministicKeys`